src/App.test.jsx

```diff
@@ -0,0 +1,59 @@
+import { render, fireEvent } from '@testing-library/react';
+
+import App from './App';
+
+describe('App', () => {
+  const renderApp = () => (
+    render(
+      <App />,
+    )
```

Comment:
```suggestion
    render((
      <App />
    ))
```

괄호를 두번 감싸주면 콤마를 없앨 수 있어요 ;)

Reply:
같은 리뷰를 받은 적이 있는데 같은 실수를 반복하네요;;

Reply:
보통 반복되는 부분에 또 리뷰를 달지는 않아요.
하나의 리뷰를 받으면 해당하는 곳이 또 있는지 전체적으로 찾아보시면 좋아요.